inference-engine/src/mkldnn_plugin/mkldnn_memory.cpp

```diff
@@ -304,6 +303,24 @@ memory::format MKLDNNMemory::GetPlainFormat(memory::dims dims) {
     }
 }
 
+memory::format MKLDNNMemory::GetPerChannelFormat(memory::dims dims) {
```

Comment:
Seems like is not used anymore

Reply:
Agree. Removed in 678d3cc5f114f602e6043713672dea16b4716e48.